Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ o2_add_dpl_workflow(hf-lc-candidate-selector
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
                     COMPONENT_NAME Analysis)
 
+o2_add_dpl_workflow(hf-jpsi-candidate-selector   
+                    SOURCES HFJpsiToEECandidateSelector.cxx
```

Comment:
The decay channel name is missing in the name of the executable:
`hf-jpsi-candidate-selector` -> `hf-jpsi-toee-candidate-selector`